dotcom-rendering/src/components/Badge.tsx

```diff
@@ -55,7 +55,7 @@ export const Badge = ({
 			data-link-name={ophanComponentLink}
 			data-component={ophanComponentName}
 			css={badgeLink}
-			role="button"
+			role={isInLabsSection ? 'none' : 'button'}
```

Comment:
I think we can still remove this line, as this doesn't look like a button whether this is in a Labs section or not

Reply:
I thought because there was already a button role some of the badges are identified as buttons but thank you for sharing screenshots for examples that shows badges in non Labs context. Addressed 7ed90cf90dd6ca09aa63b0a952a64f7eca81ac06